src/Raven.Server/Documents/Indexes/IndexStorage.cs

```diff
@@ -143,10 +144,12 @@ private unsafe void CreateSchema()
                 void PersistConfiguration()
                 {
                     var configurationTree = tx.InnerTransaction.CreateTree(IndexSchema.ConfigurationTree);
-
+                    PersistSearchEngine(configurationTree, nameof(SearchEngineType), 
```

Comment:
Should it follow the naming convention of below methods and be named as `AssertAndPersistSearchEngine()` ?